recipe_scrapers/forksoverknives.py

```diff
@@ -1,13 +1,44 @@
 # mypy: disallow_untyped_defs=False
 from ._abstract import AbstractScraper
 from ._utils import normalize_string
+from ._schemaorg import SchemaOrg
+from typing import Dict, List, Optional, Tuple, Union
+
+import re
+
+class fok_Schema(SchemaOrg):
+    def __init__(self, page_data, raw=False):
+        super().__init__(page_data)
+
+    def category(self):
+        cat = super().category()
+        r = re.compile('.*<a href="\S+">(.*)</a>')
```

Comment:
```suggestion
        r = re.compile('.*<a href="\S+">(.+)</a>')
```

(replacing `.*` instead of `.+` here, should, I think, still permit matching while reducing the maximum size of the match)